### PR TITLE
Excel Automation using Python.

### PR DESCRIPTION
**Pull request by RoopakMallik**
This solves issue #155  
Excel Automation using Python and Openpyxl Library.

### CODE REVIEW

File: Web Scrapper Bot/Readme.md
Comment (JigarJoshi04) [inline Web Scrapper Bot/Readme.md:3]: Please add a requirements.txt file.
You can use `pip freeze > requirements.txt` to generate requirement file.
And we can remove this from readme after that

File: Excel_Automation_Python/main.py
Comment (JigarJoshi04) [inline Excel_Automation_Python/main.py:35]: Why are we hardcoding data for a bot?

**Review by JigarJoshi04 — CHANGES_REQUESTED** [review]
The PR should follow a Single responsibility principle (A PR should complete only one task at a time). There should be 2 PRs one for the excel automation and the other for the web scrapper bot as they bot are independent changes.

The code written by the author of this PR, seems to have lot of harcoded values and the code written is highly dependent on the hardcoded test data. The bot should be more genric, and the test case (test data) should be independent of the actual implementation of logic.

The issue attached doesn't contain the scope or requirement in detail, please add the requirements in more elaborate way. 

Requesting maintainers to do the changes before merging the PR.
Thanks

**Comment by JigarJoshi04** [comment]
If author has any issues or need help, please feel free to ping me on Discord, Linkedin or reply to this PR.
Thanks

**Comment by RoopakMallik** [comment]
I have made the required changes in the web scrapping bot and have added a
requirement.txt file as instructed.
Please check if any further changes are required.

**Comment by JigarJoshi04** [comment]
Please check my comment https://github.com/neelshah2409/Bot-Collection/pull/158#pullrequestreview-1448237663
This PR needs those changes before getting merged.

**Comment by RoopakMallik** [comment]
I will fix the issues regarding the Excel Automation. Meanwhile please let
me know if the web scrapping bot needs any more changes.

**Comment by samanth2012** [comment]
#155 sir i was the first to rise this issue

**Pull request merged**
